github/commitstatus.go

```diff
@@ -122,7 +127,7 @@ func (s CommitStatus) Add(sha, state, targetURL, description string) error {
 	OAuthInfo := fmt.Sprintf("X-Accepted-Oauth-Scopes: %v, X-Oauth-Scopes: %v",
 		XAcceptedOauthScope, XOauthScopes)
 
-	respBody, _ := ioutil.ReadAll(resp.Body)
+	respBody, _ := io.ReadAll(resp.Body)
```

Comment:
What is the difference here? I'm quite confused as it seems there are equivalent methods to do the same.

Reply:
> What is the difference here? I'm quite confused as it seems there are equivalent methods to do the same.

Ok, I got the answer, you comment was not shown while reviewing